reStream.sh

```diff
@@ -3,10 +3,23 @@
 # these are probably the only two parameters you need to change
 ssh_host="root@10.11.99.1" # location of the remarkable
 landscape=true             # default vertical
-if [ "$1" = "-p" ]         # call with -p if you want vertical
-then
-    landscape=false
-fi
+
+# Loop through arguments and process them
+for arg in "$@"
+do
+    case $arg in
+        -p|--portrait)
+        landscape=false
+        shift
+        ;;
+        -h|--host)
```

Comment:
-h might be a bit irritating

Reply:
Hmm yes, it could cause confusion with `--help`. Do you have suggestions for another abbreviation?

Reply:
maybe `-s` for `--ssh`, or `-r` for `--remote`

Reply:
Maybe `-d|--destination`?

Reply:
You can choose :)